app/Http/Controllers/ApiController.php

```diff
@@ -253,4 +253,18 @@ public function gdprExport()
 
         return $data;
     }
+
+    public function discordVerifyMember($userId): JsonResponse
```

Comment:
Since this is an publically available endpoint I think it should have some kind of secret/bearer token so you can not just try all discord combinations against our DB

Reply:
Yup, good catch